submissions/adeyemiJuwon/phpDatabase/orders.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+include('connect.php');
+
+try{
+     $sql = 'SELECT order_items.unit_price, 
+            order_items.total_amount, orders.order_date, orders.id ,customers.street_address, customers.city,customers.state, 
+            customers.country,order_items.quantity, products.name
+            FROM order_items 
+            LEFT JOIN orders ON orders.id = order_items.order_id
+            LEFT JOIN customers on orders.customer_id = customers.id
+            LEFT JOIN products ON products.id = order_items.product_id 
+            WHERE  orders.customer_id = order_items.id';
```

Comment:
I expected you to get the `id` you are passing on the `view` button then do something like `WHERE  orders.customer_id = that id you are getting via a GET request`

Reply:
And also you are expected to hit the DB only when cookie is not set

Reply:
Okay 
Done
Thank you
@adaugochi

Reply:
@adaugochi 
Good morning.
Everything is working perfectly now